packages/ui/bin/www.js

```diff
@@ -4,7 +4,10 @@
  */
 
 const path = require("path");
-const app = require("../../ui-server")(path.join(__dirname, "../../ui-gui/build")); // eslint-disable-line
+const STATIC = path.existsSync(path.join(__dirname, "../../ui-gui/build"))
```

Comment:
@rishabh3112 `path.existsSync` doesn't exists, you need to use `fs.existsSync`

Reply:
Ohh my bad.